arduino-ide-extension/src/browser/survey/survey-retriever.ts

```diff
@@ -0,0 +1,20 @@
+import { injectable } from '@theia/core/shared/inversify';
+
+export type Survey = {
+  url: URL;
+  id: string;
+};
+
+const SURVEY_BASE_URL = 'https://surveys.hotjar.com/';
+const surveyId = '17887b40-e1f0-4bd6-b9f0-a37f229ccd8b';
+
+@injectable()
+export class SurveyRetriever {
+  public async getSurvey(): Promise<Survey> {
+    const survey: Survey = {
+      url: new URL(SURVEY_BASE_URL + surveyId),
+      id: surveyId,
+    };
+    return new Promise((resolve) => setTimeout(() => resolve(survey), 1000));
+  }
+}
```

Comment:
I would move this to the other class. This fine-grained service splitting does not have any advantage.

Reply:
@kittaakos this service should use hotjar api to get the url and ID of a survey, so eventually it won't be hardcoded. This is just code we draft because we were waiting for feedback about how to retrieve that data. (@francescospissu maybe it's better to move this PR to **Draft**?)

Reply:
@91volt and I have investigated if it was possible to retrieve the survey data using Hotjar API and unfortunately we haven't found a way to do it. We can apply the suggestion from @kittaakos. @AlbyIanna, I moved the PR to draft.

Reply:
yeah in that case having a service is not very useful 🙏